doc/openshift-monitoring.adoc

```diff
@@ -36,16 +36,16 @@ spec:
         app: example-app
     endpoints:
     - interval: 30s
-        path: /metrics
-        port: 9080-tcp
```

Comment:
The code snippet shows as :
```
apiVersion: monitoring.coreos.com/v1
kind: ServiceMonitor
metadata:
    name: myapp-monitor
    labels:
    k8s-app: myapp-monitor
    namespace: myapp
spec:
    selector:
    matchLabels:
        app: example-app
    endpoints:
    - interval: 30s
      path: /metrics
      port: 9080-tcp
```      
Encountered issue:
`error: error parsing promSM.yaml: error converting YAML to JSON: yaml: line 13: mapping values are not allowed in this context`

Formatting like instead worked for me:
the `interval` `path` and `port` were moved over and the `label` was moved over for proper formatting.
```
apiVersion: monitoring.coreos.com/v1
kind: ServiceMonitor
metadata:
    name: myapp-monitor
    labels:
      k8s-app: myapp-monitor
    namespace: myapp
spec:
    selector:
    matchLabels:
        app: example-app
    endpoints:
      - interval: 30s
        path: /metrics
        port: 9080-tcp
```